### ISSUE TITLE
Accessibility Review: SiteNow Image Gallery

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#1104]
## Problem

For the SiteNow Image Gallery, we are using the Colorbox JS library to implement  lightbox/modal type functionality. Out of the box, there were some accessibility concerns within the community, but was recommended on Drupal.org as the way to go and is actually being used on Drupal.org content.

## Proposed Solution

There is an unapproved pull request with accessibility improvements available, so we have moved forward with implementing this functionality using the colorbox library and are locally pulling the pull request locally for testing. Demo: https://sitenowdemo.drupal.uiowa.edu/about

The PR attempts to address:
- Add aria-hidden attributes on the dialog box and navigational buttons.
- Update the aria-hidden value to true or false at the appropriate times.
- Add aria-label attributes to navigational buttons using the colorbox default settings at page load time and then updating the value based on user-settings when a colorbox is loaded.
- Add aria-labelledby and aria-describedby attributes to the div containing role='dialog'.
- Reworked focus trapping: now the dialog itself isn't focusable and tab/shift+tab cycles through the elements in the dialog
- ARIA live regions are used to announce changes in the title and the dialog content

## Test

@inspector508, could you look over the image gallery functionality we have implemented so far and post your analysis in the this issue's comments?

**Comment by joewhitsitt** [comment]
Pasting bits of Todd's review:

```
Here’s what I can do with that page using only a screen reader and the keyboard:

- Load page
- Browse headings/Skip to Content/Traverse
- Move from image to image (with ALT text, I think)
- Find “kitten”
- Save kitten to file system

Without the screen reader (as a visual keyboard user), I cannot visually track the cursor location as I move through the images and therefore cannot recognize when I arrive at the kitten. A somewhat more prominent focus style would be useful. I think a principled case could be made for applying a hover state as well, especially if it made the interface more appealing in general.

```
